Formula/python@3.10.rb

```diff
@@ -253,7 +266,7 @@ def install
       --no-build-isolation
     ]
     whl_build = buildpath/"whl_build"
-    system bin/"python3", "-m", "venv", whl_build
+    system bin/"python#{version.major_minor}", "-m", "venv", whl_build
```

Comment:
@XuehaiPan suggested we do
```ruby
def python3
  bin/"python#{version.major_minor}"
end
```
which is a good idea, given the number of times this pattern appears now.